adapters/vendors/rabbitmq/rabbit.py

```diff
@@ -141,6 +140,7 @@ def main(self):
                     print("Unable to send event to RabbitMQ because: %s" % err)
                     print("The following event will be retried: %r" % buffer)
                     self.rabbit_conn()
+                sys.stdout.flush()
```

Comment:
At somestage (during setting the adapter up the first time) the program would crash (typically when variables are set incorrectly iirc) but it wouldn't print any info.
So this could be removed if necessary.